src/script/user/UserMapper.js

```diff
@@ -134,7 +134,16 @@ export default class UserMapper {
       userEntity.managedBy(managed_by);
     }
 
-    userEntity.extendedFields(extended_fields);
+    userEntity.extendedFields(
+      extended_fields || {
+        /* eslint-disable sort-keys */
+        Title: 'Chief Design Officer',
+        Phone: ' 0172 987 65 43',
+        Entity: 'Orange/OBS/EQUANT/CSO/IBO/OEC/SERVICE OP/CS MGT/CSM EEMEA',
+        Email: 'michelle@acme.com',
+        /* eslint-enable sort-keys */
```

Comment:
does the sort order matters here?! Why disabling?

Reply:
Yes, this is mock data that should be displayed in exactly this order.